setup.py

```diff
@@ -36,6 +36,7 @@ def package_files(directory):
 extra_deps['dev'] = [
     'junitparser>=2.1.1',
     'coverage[toml]>=6.1.1',
+    'fasteners>=0.16.3',  # run_directory_uploader tests require fasteners
```

Comment:
I think it may make sense to split off all tests dependencies into its own category now that we have a couple of them

Reply:
We could split out `dev` into `tests` and `docs`? Would prefer to do in a separate PR...

Reply:
See #129